boards/beagle/pocketbeagle_2/pocketbeagle_2_am6232_a53-pinctrl.dtsi

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2025 Ayush Singh, BeagleBoard.org Foundation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/ti-k3-pinctrl.h>
+
+&pinctrl {
+	main_uart6_rx_default: main_uart6_rx_default {
+		pinmux = <K3_PINMUX(0x001c, PIN_INPUT, MUX_MODE_3)>;
+	};
+
+	main_uart6_tx_default: main_uart6_tx_default {
+		pinmux = <K3_PINMUX(0x0020, PIN_OUTPUT, MUX_MODE_3)>;
```

Comment:
```suggestion
		pinmux = <K3_PINMUX(0x0198, PIN_OUTPUT, MUX_MODE_3)>;
```

Reply:
Umm, the uart6 pins used in debug port are J23 and J25, with SW flow control. So the pins defined here are correct.

You can also check the linux devicetree [here](https://github.com/beagleboard/BeagleBoard-DeviceTrees/blob/c96961e2b333b6cb629d49c147718af202ff6090/src/arm64/ti/k3-am6232-pocketbeagle2.dts#L230).

![2025-04-28T16:55:16,141181937+05:30](https://github.com/user-attachments/assets/3900e71e-d7ca-4bec-8078-ff76c3191cc2)

Reply:
Ah yes, maybe we should add the pin and muxing as comment next to it similar to Linux.

Reply:
Yes, please add the ball number.